### PR TITLE
Add testing against jammy builder

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

Part of https://github.com/paketo-buildpacks/nodejs/issues/607

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by ryanmoran** [comment]
This seems to be relying upon the work done in https://github.com/paketo-buildpacks/github-config/pull/503. Are we supposed to wait until that gets merged so that it doesn't get wiped out by the automation?

**Comment by sophiewigmore** [comment]
@ryanmoran I forgot to merge in the Github Config PR earlier, but I just did. It shouldn't make a difference now if this PR goes in first, or the `update github-config` PR goes in.

**Review by ryanmoran — APPROVED** [review]

**Pull request merged**
